src/content/docs/data-apis/ingest-apis/event-api/introduction-event-api.mdx

```diff
@@ -257,29 +257,20 @@ The Event API accepts specific formats for attributes included in the payload. O
 
-    * Payload total size: **1MB(10^6 bytes) maximum per POST**. We highly recommend using compression.
-
-    * The payload must be encoded as **UTF-8**.
-
-    * Maximum number of attributes per event: 255
-
-    * Maximum length of attribute name: 255 characters
-
-    * Maximum length of attribute value: 4096 characters
-
-    * There are rate [limits on the number of HTTP requests per minute](#post-limit) sent to the Event API.
-
-      Some specific attributes have additional restrictions:
-
-    * `accountId`: This is a reserved attribute name. If it is included, it will be dropped during ingest.
-
-    * `entity.guid`, `entity.name`, and `entity.type`: These attributes are used internally to identify entities. Any values submitted with these keys in the attributes section of a metric data point may cause undefined behavior such as missing entities in the UI or telemetry not associating with the expected entities. For more information please refer to [Entity synthesis](/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic/#entity-synthesis).
-
-    * [`appId`](/docs/apis/rest-api-v2/requirements/find-product-id#apm): Value must be an integer. If it is not an integer, the attribute name and value will be dropped during ingest.
-
-    * `eventType`: Can be a combination of alphanumeric characters, `_` underscores, and `:` colons.
-
-    * `timestamp`: Must be a Unix epoch timestamp. You can define timestamps either in seconds or in milliseconds.
+      * Payload total size: **1MB(10^6 bytes) maximum per POST**. We highly recommend using compression.
+      * The payload must be encoded as **UTF-8**.
+      * Maximum number of attributes per event: 255
+      * Maximum length of attribute name: 255 characters
+      * Maximum length of attribute value: 4096 characters
+      * There are rate [limits on the number of HTTP requests per minute](#post-limit) sent to the Event API.
+
+    Some specific attributes have additional restrictions:
+
+      * `accountId`: This is a reserved attribute name. If it is included, it will be dropped during ingest.
+      * `entity.guid`, `entity.name`, and `entity.type`: These attributes are used internally to identify entities. Any values submitted with these keys in the attributes section of a metric data point may cause undefined behavior such as missing entities in the UI or telemetry not associating with the expected entities. For more information please refer to [Entity synthesis](/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic/#entity-synthesis).
+      * [`appId`](/docs/apis/rest-api-v2/requirements/find-product-id#apm): Value must be an integer. If it is not an integer, the attribute name and value will be dropped during ingest.
+      * `eventType`: Can be a combination of alphanumeric characters, `_` underscores, and `:` colons.
+      * `timestamp`: Must be a Unix epoch timestamp. You can define timestamps either in seconds or in milliseconds.
```

Comment:
```suggestion
      * `accountId`: This is a reserved attribute name. If it is included, it will be dropped during ingest.
      * `entity.guid`, `entity.name`, and `entity.type`: These attributes are used internally to identify entities. Any values submitted with these keys in the attributes section of a metric data point may cause undefined behavior such as missing entities in the UI or telemetry not associating with the expected entities. For more information please refer to [Entity synthesis](/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic/#entity-synthesis).
      * [`appId`](/docs/apis/rest-api-v2/requirements/find-product-id#apm): Value must be an integer. If it is not an integer, the attribute name and value will be dropped during ingest.
      * `eventType`: This attribute can be a combination of alphanumeric characters, `_` underscores, and `:` colons.
      * `timestamp`: This attribute must be a Unix epoch timestamp, defined in either seconds or milliseconds.
```